pom.xml

```diff
@@ -93,8 +93,9 @@ under the License.
     <gpg.useagent>true</gpg.useagent>
     <minimalMavenBuildVersion>3.6.3</minimalMavenBuildVersion>
     <minimalJavaBuildVersion>1.8</minimalJavaBuildVersion>
+    <targetJavaVersion></targetJavaVersion>
     <maven.compiler.source>${maven.compiler.target}</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.target>7</maven.compiler.target><!-- use version numbers without the "1." prefix (supported since javac 8) -->
```

Comment:
should we enforce with https://maven.apache.org/enforcer/enforcer-rules/requireProperty.html that this property value does no longer start with "1."?

Reply:
I'm not sure if `7` is supported .... maybe it is time to switch it to `8`

Reply:
yes it is: back to 5 if javac 8 is used, https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html#BHCDIFEE

Reply:
javac 17 still supports 7, only javac21 only supports 8+.